services/iam/doc/openapi.json

```diff
@@ -1105,6 +1105,23 @@
           }
         }
       },
+      "MutableRole": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "type": "string"
+          },
+          "permissions": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/Permissions"
+            }
+          },
+          "scope": {
```

Comment:
If you look at the model and the routes for the Roles, you will see which elements are actually accepted when updating or creating the role.

Example:
https://github.com/openintegrationhub/openintegrationhub/blob/b5dda021caa0e291169d34dce00266c1f74714dd/services/iam/src/routes/roles.js#L102-L107

As you can see in the snippet from the POST router here, the scope is not accepted in the request. I am not sure what scope is used for in roles, and this should probably become a separate issue. 'description' and 'isGlobal' however, are accepted.

Reply:
It exists in the membership as well! I am not sure where it is needed though!